components/event-sources/reconciler/object/service.go

```diff
@@ -17,18 +17,14 @@ limitations under the License.
 package object
 
 import (
-	"strconv"
-
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"knative.dev/serving/pkg/apis/autoscaling"
-	servingv1alpha1 "knative.dev/serving/pkg/apis/serving/v1alpha1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-// NewService creates a Service object.
-func NewService(ns, name string, opts ...ObjectOption) *servingv1alpha1.Service {
-	s := &servingv1alpha1.Service{
+// NewDeployment creates a Service object.
```

Comment:
Method name is wrong:

```suggestion
// NewService creates a Service object.
```